packages/shared/src/components/referral/SearchReferralButton.tsx

```diff
@@ -42,7 +42,7 @@ export function SearchReferralButton({
     <button
       type="button"
       className={classNames(
-        'flex flex-row items-center justify-center rounded-12 py-1 px-3 font-bold text-theme-label-tertiary bg-theme-overlay-from',
+        'flex flex-row items-center justify-center typo-callout rounded-12 py-1 px-3 font-bold text-theme-label-tertiary bg-theme-overlay-from',
```

Comment:
Can `font-bold` be removed?
```suggestion
        'flex flex-row items-center justify-center typo-callout rounded-12 py-1 px-3 text-theme-label-tertiary bg-theme-overlay-from',
```

Reply:
Nvm, `typo-callout` doesn't have bold

Reply:
Yep, also, if I remember correctly, I think only `typo-headline` provides `font-bold` out of the box.